src/Bundle/Maker/MakeFactory/FactoryFinder.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Zenstruck\Foundry\Bundle\Maker\MakeFactory;
+
+use Zenstruck\Foundry\ModelFactory;
+
+final class FactoryFinder
```

Comment:
not sure of this name... it's more like an index of factory classes

Reply:
`FactoryRegistry`?

Reply:
I'm not sure... it does not even contain actual factory, but just a map with [factory class => object class]

Reply:
Fair, we can always rename later as it's internal